### DISCUSSION TITLE
supported file question: are sketchfab free models viewable with this script?

### DISCUSSION THREAD

**Discussion by inglesuniversal** [google/model-viewer#2192] [Q&A]
thanks

**Comment by hybridherbst** [comment]
Depends on model complexity and usage of transparency etc., works in theory.
See https://github.com/google/model-viewer/issues/1065 for a model that is directly from Sketchfab.

**Comment by cdata** [comment]
@inglesuniversal Theoretically, yes. But, keep in mind that `<model-viewer>` only supports the glTF (`.gltf`/`.glb`) file format. In the case of Sketchfab, you can download most models as glTF, but in many cases those glTFs are automatically generated from a source file in some other format. There are three main considerations you should keep in mind when comparing models as you see them on Sketchfab compared to `<model-viewer>`:

 1. Sketchfab has post-processes that are not yet available in `<model-viewer>`. If you identify any of these that you want, please file a feature request with us!
 2. Sketchfab supports material types and properties that are not supported in the glTF file format. If you identify any of these that you want, please make a feature request of [glTF](https://github.com/KhronosGroup/glTF/issues).
 3. Automatically-generated models have a relatively high chance of including artifacts or mistakes compared to the source model that they are based on. YMMV when downloading such glTF files.

**Reply by inglesuniversal** [reply]
Hi All, any update around the ability to import Sketchfab models into the newest Model-Viewer update?

**Reply by hybridherbst** [reply]
You can download glTF files from Sketchfab and load them with model-viewer (already when you asked the original question).
Do you mean something else?

**Comment by inglesuniversal** [comment]
Thanks for your prompt reply. Let my try again. It’s been such a long time.
